edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/ConnectionManager.cs

```diff
@@ -75,7 +75,7 @@ await currentDeviceConnection
         public Task RemoveDeviceConnection(string id)
         {
             return this.devices.TryGetValue(Preconditions.CheckNonWhiteSpace(id, nameof(id)), out ConnectedDevice device)
-                ? this.RemoveDeviceConnection(device, false)
+                ? this.RemoveDeviceConnection(device, true)
```

Comment:
Let's remove the flag altogether..

Reply:
will do in another PR.

Reply:
Actually, let's add a setting (env var) to EdgeHub close the cloud connection when device is gone or not. Default will be to close the cloud connection, but the flag lets you turn it off.

Reply:
1. In what case we would like to keep the cloud connection even the module is removed from deployment?
2. There are totally 5 usage references and 2 of them passed false in 2nd argument.  One is this usage.  @varunpuranik  I would like to clarify, does env var setting to control all 5 usages or only those usages currently passing false as 2nd argument?